CSharpOOP/CSharpOOP/Program.cs

```diff
@@ -0,0 +1,179 @@
+//Console.WriteLine(receipt);
+#endregion
+
+#region Task_4
+//Console.WriteLine("Enter side 1 length: ");
+//double side1 = double.Parse(Console.ReadLine());
+
+//Console.WriteLine("Enter side 2 length: ");
+//double side2 = double.Parse(Console.ReadLine());
+
+//Console.WriteLine("Enter side 3 length: ");
+//double side3 = double.Parse(Console.ReadLine());
+
+//Console.WriteLine("Enter side 4 length: ");
+//double side4 = double.Parse(Console.ReadLine());
+
+//Console.WriteLine("Enter side 5 length: ");
+//double side5 = double.Parse(Console.ReadLine());
+
+//double trianglePerimeter = new Homework_1().CalculatePerimeter(side1, side2, side3);
+//double rectanglePerimeter = new Homework_1().CalculatePerimeter(side1, side2, side3, side4);
+//double pentagonPerimeter = new Homework_1().CalculatePerimeter(side1, side2, side3, side4, side5);
+
+//Console.WriteLine($"Triangle perimeter is {trianglePerimeter}");
+//Console.WriteLine($"Rectangle perimeter is {rectanglePerimeter}");
+//Console.WriteLine($"Pentagon perimeter is {pentagonPerimeter}");
+
+#endregion
+
+#region Task_5
+//Console.WriteLine("Enter a side length of the triangle: ");
+//double sideLength = Convert.ToDouble(Console.ReadLine());
+
+//Console.WriteLine("Enter the triangle height: ");
+//double height = Convert.ToDouble(Console.ReadLine());
+
+//Console.WriteLine("Enter the circle radius: ");
+//double radius = Convert.ToDouble(Console.ReadLine());
+
+//double triangleArea = new Homework_1().CalculateArea(sideLength, height);
+//Console.WriteLine($"Area of the triangle is {triangleArea}");
+
+//double circleArea = new Homework_1().CalculateArea(radius);
+//Console.WriteLine($"Area of the circle is {circleArea}");
+#endregion
+
+#region Task_6
+/* Encapsulation: A bank account hides information about the account and transactions, providing access only through specific operations.
+ * Inheritance: Different animal species inherit common characteristics and properties from their parent species.
+ * Polymorphism: A media player has a "play" method that can work with different types of media files.
+ * Abstraction: Mobile phone users can use an interface to interact with some of the functions, but it's not necessary to understand the whole complexity of software and hardware
+ */
+#endregion
+
+class Homework_1
+{
+    public double CalculateTotalAmount(int quantity, double unitPrice, double discount)
+    {
+        double price = quantity * unitPrice;
+        double discountFraction = discount / 100;
+        double amountWithDiscount = price * (1 - discountFraction);
+        return amountWithDiscount;
+    }
+
+    public string CreateReceipt(string itemName1, string itemName2, string itemName3, double unitPrice1, double unitPrice2, double unitPrice3, double discount, int quantity1 = 2, int quantity2 = 2, int quantity3 = 2)
```

Comment:
This method is not reusable :)
suggestion: use method with 1 param of all types and create receipt in mail part :)